docs/cookbook/modifying-a-template-in-administration.md

```diff
@@ -0,0 +1,133 @@
+# Modifying a Template in Administration
+
+In this tutorial, we'll show you how to proceed if you need to modify a twig template in the administration.
+We will demonstrate this procedure with the example How to display a product transfer status on the product detail page in the administration.
+
+## Adding the information into the template
+
+This tutorial assumes that the product already contains the transfer status attribute.
+If you need to extend the product with a new attribute, see the tutorial [Adding new attribute to an entity](./adding-new-attribute-to-an-entity.md).
```

Comment:
In my opinion this tutorial should not be dependent on any other cookbook that does not even add the same attribute to entity.

Even if you do everything in [Adding new attribute to an entity], this cookbook will not work for you instantly. I think thats little bit sad and cookbooks should be really functional out of the batch.

I would not show example about not existing attribute, even the US says that it could be only text under the form. Something like "I overwritten template, yey!"

Importatnt for me is to have cookbook that will be functional after completing all tasks, what do you think?

Reply:
In my opinion this tutorial should not be dependent on any other cookbook that does not even add the same attribute to entity.
- consulted with our architect, he liked this approach

Even if you do everything in [Adding new attribute to an entity], this cookbook will not work for you instantly. I think thats little bit sad and cookbooks should be really functional out of the batch.
- why do you think it will not work?

I would not show example about not existing attribute, even the US says that it could be only text under the form. Something like "I overwritten template, yey!"
- I decided to make a cookbook for some real problem. This specific problem - product trasfer status - was based on a real US from white. During the PIA, this intention was fully consulted with PH and approved.

Importatnt for me is to have cookbook that will be functional after completing all tasks, what do you think?
- definitelly agree with you. And it is functional after completing all tasks. Or not? Can you mention here the missing tasks?

Reply:
Okay Ill try to explain it more.

Imagine new developer with clean project-base, he just wants to learn few things about shopsys so he starts our cookbooks, he finish the [Adding new attribute to an entity] and finish this Cookbook, even if he copy every code you say and do exactly what you did, it Will not work. Because it is built around `isTransfered` attribute which nobody said developer to implement.

So for Me there are 2 scenarios to go.
1) Built it around existing properties or even without properties.
   - This means that developer can finish this cookbook without doing the other ones (Imagine that someone just wants to find out how to overwrite template, he does not care about extending entity or anything like that)
2) Built it about extended properties, BUT use `extId` instead, so you do not have to implement `isTransfered` property by yourself just to make it work.

@PetrHeinz ?? 

Do you understand what is my point? If you and Peter do not think its important, fine then but for me it is important

Reply:
I applied the rule of the third person if two people disagree with each other. @vitek-rostislav has gone through this cookbook and he is ok with this solution with transfer status attribute.

Second thing, I still don't understand what do you mean with "If you and Peter do not think its important, fine then but for me it is important".